packages/connect/src/connectors/ecosystem/ecosystemWallet.ts

```diff
@@ -0,0 +1,135 @@
+import { ethers } from 'ethers'
+import { getAddress } from 'viem'
+import { createConnector, Connector } from 'wagmi'
+
+import { normalizeChainId } from '../../utils/helpers'
+
+import { EcosystemWalletTransportProvider } from './provider'
+
+export interface EcosystemConnector extends Connector {
+  type: 'ecosystem-wallet'
+  auxData?: Record<string, unknown>
+}
+
+export interface BaseEcosystemConnectorOptions {
+  projectAccessKey: string
+  walletUrl: string
+  defaultNetwork: number
+  isDev?: boolean
+}
+
+ecosystemWallet.type = 'ecosystem-wallet' as const
+
+export function ecosystemWallet(params: BaseEcosystemConnectorOptions) {
+  type Provider = EcosystemWalletTransportProvider
+  type Properties = {
+    ecosystemProvider: EcosystemWalletTransportProvider
+    auxData?: Record<string, unknown>
+  }
+  type StorageItem = {}
+
+  const isDev = !!params?.isDev
+  const nodesUrl = isDev ? 'https://dev-nodes.sequence.app' : 'https://nodes.sequence.app'
```

Comment:
Should this be a url overwrite instead?
```suggestion
  const nodesUrl = params.nodesUrl ?? 'https://nodes.sequence.app'
```